Packs/Jira/Scripts/JiraAddComment/JiraAddComment.yml

```diff
@@ -0,0 +1,34 @@
+args:
+- description: comment to be added to the Jira issue.
+  name: comment
+  required: true
+- auto: PREDEFINED
+  defaultValue: comment tag
+  description: The comment tag. Use the comment entry tag to Jira configured in your instance to mirror
+    the comment to Jira.
+  name: tags
+  predefined:
+  - comment tag
```

Comment:
Because the user can enter a different tag, I set the default value to be equal to the default value in the instance configuration, but the user has the option to enter a different one if they chose to.